src/tabs/__tests__/tabs.test.tsx

```diff
@@ -33,29 +33,17 @@ function wrap(container: HTMLElement) {
 }
 
 function pressRight(wrapper: TabsWrapper) {
-  act(() => {
```

Comment:
Small drive by simplification. After https://github.com/cloudscape-design/test-utils/pull/25, no manual wrapping is needed